### PR TITLE
Fix various minor typos

### PR DESCRIPTION
**Pull request by melquiadess**
A few of typos fixed, based on IntelliJ inspection

### CODE REVIEW

**Review by skyace65 — CHANGES_REQUESTED** [review]
Two minor changes need to be made but the rest of this looks good. When updating this PR please keep everything to 1 commit. If you don't know how to do that we have a guide [here](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html).

File: tutorials/3d/physical_light_and_camera_units.rst [hunk 1]
```diff
@@ -252,7 +252,7 @@ more cinematic appearance.
 
 Common focal length values used in filmmaking and photography are:
 
-- **Fisheye (ultrawide angle):** Below 15 mm. Nearly no depth of field visible.
+- **Fisheye (ultra-wide angle):** Below 15 mm. Nearly no depth of field visible.
```
[the remark this answers is not in the document]
Comment (melquiadess) [reply tutorials/3d/physical_light_and_camera_units.rst]: Fair enough, reverted

**Review by skyace65 — APPROVED** [review]

File: tutorials/navigation/navigation_using_navigationobstacles.rst [hunk 2]
```diff
@@ -49,7 +49,7 @@ Ideally when an obstacle is moving the static vertices are removed and instead t
 Similar to agents the obstacles can make use of the ``avoidance_layers`` bitmask.
 All agents with a matching bit on their own avoidance mask will avoid the obstacle.
 
-Procedual obstacles
+Procedural obstacles
 ~~~~~~~~~~~~~~~~~~~
```
Comment (skyace65) [inline tutorials/navigation/navigation_using_navigationobstacles.rst]:
```suggestion
~~~~~~~~~~~~~~~~~~~~
```

Missed this before, my bad. Underline length needs to be adjusted so it matches up with the section title.

**Review by skyace65 — CHANGES_REQUESTED** [review]

File: tutorials/navigation/navigation_using_navigationobstacles.rst (hunk 2 above)
Comment (melquiadess) [reply tutorials/navigation/navigation_using_navigationobstacles.rst]: I'd missed too :) fixed

File: tutorials/3d/occlusion_culling.rst [hunk 3]
```diff
@@ -85,7 +85,7 @@ In Godot, occlusion culling works by rasterizing the scene's occluder geometry
 to a low-resolution buffer on the CPU. This is done using
 the software raytracing library `Embree <https://github.com/embree/embree>`__.
 
-The engine then uses this low-resolution buffer to test occludees'
+The engine then uses this low-resolution buffer to test occludee's
```
Comment (AThousandShips) [inline tutorials/3d/occlusion_culling.rst:88]:
```suggestion
The engine then uses this low-resolution buffer to test the occludee's
```

Comment (melquiadess) [reply tutorials/3d/occlusion_culling.rst]: done

**Review by AThousandShips — APPROVED** [review]

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Merged. Thanks and congrats on your first merged contribution!

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.2.

**Pull request merged**